cpp/dolfin/fem/dofs_permutation.cpp

```diff
@@ -35,155 +35,208 @@ int get_num_permutations(mesh::CellType cell_type)
   default:
     LOG(WARNING) << "Dof permutations are not defined for this cell type. High "
                     "order elements may be incorrect.";
-    return 0;
+    return -1;
   }
 }
 //-----------------------------------------------------------------------------
-/// Calculates the number of times the rotation and reflection of a triangle
-/// should be applied to a triangle with the given global vertex numbers
-/// @param[in] v1, v2, v3 The global vertex numbers of the triangle's vertices
+/// Returns an empty array
+/// @param[in] vs The global vertex number of the point's vertex
+/// @return An empty array
+Eigen::Array<std::int8_t, 1, Eigen::Dynamic>
+calculate_point_orders(const Eigen::Array<std::int64_t, 1, 1>& vs)
+{
+  return {};
+}
+//-----------------------------------------------------------------------------
+/// Calculates whether or not an interval should be flipped
+/// @param[in] vs The global vertex numbers of the interval's vertices
+/// @return The reflection order for the interval
+Eigen::Array<std::int8_t, 1, Eigen::Dynamic>
+calculate_interval_orders(const Eigen::Array<std::int64_t, 1, 2>& vs)
```

Comment:
These function will need to take a `Eigen::Array<std::int8_t, 1, Eigen::Dynamic>` as an input argument for performance reasons. See comment below.